modules/csg/csg_shape.cpp

```diff
@@ -141,11 +141,11 @@ bool CSGShape3D::is_root_shape() const {
 }
 
 void CSGShape3D::set_snap(float p_snap) {
-	snap = p_snap;
+	snap = p_snap / 100;
```

Comment:
This breaks compatibility, is it really needed?

It's not change the way `snap` is used internally, only breaking compat for on the user side, which seems unnecessary.

If the problem was not being able to input small enough numbers, you can change the step in the range hint.